### PR TITLE
Add matterbridge (fixes #37)

### LINKED ISSUE #37: Add Matterbridge
Add and configure Matterbridge for Discord integration. This includes setup of a relay on the same server.

### PR DESCRIPTION
**Pull request by CitadelCore**
Add matterlink and its associated configuration - secret will be deployed via CI/CD.

### CODE REVIEW

File: descentfrozenhell/config/matterlink/identities.hjson [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{ 
+	/* username: NikkyAi */
```
Comment (skyem123) [inline descentfrozenhell/config/matterlink/identities.hjson]: What is this doing?

File: descentfrozenhell/config/matterlink/matterlink.hjson [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+				"explosion.player": [ 
+					"💣",
+					"💥"
+				],
+				/* ieWireShock */ 
+				"ieWireShock": [ 
+					"🔌",
+					"⚡"
+				],
+				/* immersiverailroading:hitByTrain */
```
Comment (skyem123) [inline descentfrozenhell/config/matterlink/matterlink.hjson]: We don't have immersive railroading, could change the death message for railcraft?

File: descentfrozenhell/config/matterlink/identities.hjson (hunk 1 above)
Comment (CitadelCore) [reply descentfrozenhell/config/matterlink/identities.hjson]: I just copied that file over as it was generated on starting the mod - didn't touch it.

File: descentfrozenhell/config/matterlink/matterlink.hjson (hunk 2 above)
Comment (CitadelCore) [reply descentfrozenhell/config/matterlink/matterlink.hjson]: Yeah, in that case we should find out what the death message "tag" is for railcraft. I'll take a look

Comment (CitadelCore) [reply descentfrozenhell/config/matterlink/matterlink.hjson]: Okay, added death emotes for railcraft in the config.

**Pull request merged**